src/Tribe/Linked_Posts.php

```diff
@@ -533,7 +533,8 @@ public function get_linked_post_info( $linked_post_type, $args = array(), $linke
 				'private',
 				'pending',
 			),
-			'orderby'              => 'post__in',
+			'order'                => 'ASC',
+			'orderby'              => 'post__in post_title',
```

Comment:
I know this is legit, but I am personally always more comfortable with an array for more specificity when passing multiple `orderby` values. 
I just think it's clearer?
YMMV.